package.json

```diff
@@ -120,9 +120,9 @@
     "@wpmedia/author-content-source-block": "file:blocks/author-content-source-block",
     "@wpmedia/byline-block": "file:blocks/byline-block",
     "@wpmedia/card-list-block": "file:blocks/card-list-block",
-    "@wpmedia/content-api-source-block": "file:blocks/content-api-source-block",
     "@wpmedia/collections-content-source-block": "file:blocks/collections-content-source-block",
     "@wpmedia/commerce-product-content-source-block": "file:blocks/commerce-product-content-source-block",
+    "@wpmedia/content-api-source-block": "file:blocks/content-api-source-block",
```

Comment:
deliberate change here?

Reply:
no? that's weird....i didnt think i had touched this file

Reply:
I think by saving in this file, the linter/prettier alphabetized this list

Reply:
"con" comes after "com"

Reply:
oh interesting. I wonder if you have a global prettier setup

Reply:
no worries